pkg/processor/build/test/build_test.go

```diff
@@ -31,6 +32,54 @@ type TestSuite struct {
 	httpsuite.TestSuite
 }
 
+func (suite *TestSuite) TestBuildFuncFromSourceString() {
```

Comment:
As I said in Slack, this test should happen per runtime due to how we currently test things (verify functionality for all runtimes). As such it should reside in `pkg/processor/build/runtime/test/suite/suite.go`. Rather than passing code directly you can use the same mechanism other tests use to get per-runtime code - that is, use `suite.RuntimeSuite.GetFunctionInfo` to get the path of the reverser function and then read that into a string.